tests/pulser_simulation/test_qutip_backend_v2.py

```diff
@@ -0,0 +1,136 @@
+
+Omega_max = 4 * 2 * math.pi
+U = Omega_max / 2
+delta_0 = -6 * U
+delta_f = 2 * U
+t_rise = 500
+t_fall = 1000
+t_sweep = (delta_f - delta_0) / (2 * np.pi * 10) * 1000
+
+
+@pytest.fixture
+def sequence():
+    R_interatomic = pulser.devices.MockDevice.rydberg_blockade_radius(U)
+    reg = pulser.Register.rectangle(1, 4, R_interatomic, prefix="q")
+
+    rise = pulser.Pulse.ConstantDetuning(
+        pulser.waveforms.RampWaveform(t_rise, 0.0, Omega_max), delta_0, 0.0
+    )
+    sweep = pulser.Pulse.ConstantAmplitude(
+        Omega_max,
+        pulser.waveforms.RampWaveform(t_sweep, delta_0, delta_f),
+        0.0,
+    )
+    fall = pulser.Pulse.ConstantDetuning(
+        pulser.waveforms.RampWaveform(t_fall, Omega_max, 0.0), delta_f, 0.0
+    )
+
+    seq = pulser.Sequence(reg, pulser.devices.MockDevice)
+    seq.declare_channel("ising_global", "rydberg_global")
+    seq.add(rise, "ising_global")
+    seq.add(sweep, "ising_global")
+    seq.add(fall, "ising_global")
+
+    return seq
+
+
+def test_qutip_backend_v2_energy(sequence):
+    with pytest.raises(
+        TypeError, match="'config' must be an instance of 'EmulationConfig'"
+    ):
+        QutipBackendV2(sequence, config="tralala")
+
+    config = QutipConfig(
+        default_evaluation_times=(1.0,),
+        observables=[
+            StateResult(evaluation_times=[1.0]),
+            Energy(evaluation_times=[0.001 * n for n in range(1001)]),
+        ],
+    )
+    backend = QutipBackendV2(sequence, config=config)
+    results = backend.run()
+    assert (
+        results.get_result("energy", 0.0)
+        == results.energy[0]
+        == pytest.approx(0.0)
+    )
+    assert (
+        results.get_result("energy", 1.0)
+        == results.energy[-1]
+        == pytest.approx(
+            qutip.expect(
+                backend._sim_obj.get_hamiltonian(t_rise + t_sweep + t_fall),
+                results.state[-1].to_qobj(),
+            )
+        )
+    )
+
+
+def test_qutip_backend_v2_stochastic_noise(sequence):
+    np.random.seed(123)
+
+    noise_model = pulser.noise_model.NoiseModel(
+        temperature=50.0,
+        p_false_neg=0.01,
+        amp_sigma=1e-3,
+        runs=500,
+        # FIXME: samples_per_run doesn't make sense with new API.
+        samples_per_run=1,
+    )
+
+    config = QutipConfig(
+        default_evaluation_times=(1.0,),
+        observables=[
+            StateResult(evaluation_times=[1.0]),
+            Occupation(evaluation_times=[0.001 * n for n in range(1001)]),
+        ],
+        noise_model=noise_model,
+    )
+    backend = QutipBackendV2(sequence, config=config)
+    results = backend.run()
+
+    # Same run with old API
+    sim_config = SimConfig.from_noise_model(noise_model)
+    qutip_emulator = QutipEmulator.from_sequence(sequence, config=sim_config)
+    results_old_api = qutip_emulator.run()
+
+    times = results.get_result_times("occupation")
+    occupation = np.array([x[0] for x in results.occupation])
```

Comment:
Since you are only using the first qubit, maybe you can reduce the number of qubits in the sequence. Maybe not to 1 so that you have interactions, but 2 should be fine